src/dodal/devices/i19/pin_col_stages.py

```diff
@@ -0,0 +1,169 @@
+import asyncio
+
+from bluesky.protocols import Movable
+from ophyd_async.core import AsyncStatus, StandardReadable, StrictEnum, SubsetEnum
+from ophyd_async.epics.core import epics_signal_r
+from pydantic import BaseModel
+
+from dodal.devices.i19.mapt_configuration import (
+    MAPTConfigurationControl,
+    MAPTConfigurationTable,
+)
+from dodal.devices.motors import XYStage
+from dodal.log import LOGGER
+
+_PIN = "-MO-PIN-01:"
+_COL = "-MO-COL-01:"
+_CONFIG = "-OP-PCOL-01:"
+
+
+class PinColRequest(StrictEnum):
```

Comment:
Could: This isn't being used in `ophyd-async` so can just be an `Enum` not a `StrictEnum`